gensim/topic_coherence/indirect_confirmation_measure.py

```diff
@@ -8,12 +8,12 @@
 This module contains functions to compute confirmation on a pair of words or word subsets.
 
```

Comment:
Explain how the indirect confirmation measures work and why they are useful similar to the competing car brands explanation in the paper.